datasets/conv_questions/conv_questions.py

```diff
@@ -0,0 +1,172 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+ConvQuestions is the first realistic benchmark for conversational question answering over
+knowledge graphs. It contains 11,200 conversations which can be evaluated over Wikidata.
+They are compiled from the inputs of 70 Master crowdworkers on Amazon Mechanical Turk,
+with conversations from five domains: Books, Movies, Soccer, Music, and TV Series.
+The questions feature a variety of complex question phenomena like comparisons, aggregations,
+compositionality, and temporal reasoning. Answers are grounded in Wikidata entities to enable
+fair comparison across diverse methods. The data gathering setup was kept as natural as
+possible, with the annotators selecting entities of their choice from each of the five domains,
+and formulating the entire conversation in one session. All questions in a conversation are
+from the same Turker, who also provided gold answers to the questions. For suitability to knowledge
+graphs, questions were constrained to be objective or factoid in nature, but no other restrictive
+guidelines were set. A notable property of ConvQuestions is that several questions are not
+answerable by Wikidata alone (as of September 2019), but the required facts can, for example,
+be found in the open Web or in Wikipedia. For details, please refer to our CIKM 2019 full paper
+(https://dl.acm.org/citation.cfm?id=3358016).
+"""
+
+
+import json
+import os
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@InProceedings{christmann2019look,
+  title={Look before you hop: Conversational question answering over knowledge graphs using judicious context expansion},
+  author={Christmann, Philipp and Saha Roy, Rishiraj and Abujabal, Abdalghani and Singh, Jyotsna and Weikum, Gerhard},
+  booktitle={Proceedings of the 28th ACM International Conference on Information and Knowledge Management},
+  pages={729--738},
+  year={2019}
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+ConvQuestions is the first realistic benchmark for conversational question answering over knowledge graphs.
+It contains 11,200 conversations which can be evaluated over Wikidata. The questions feature a variety of complex
+question phenomena like comparisons, aggregations, compositionality, and temporal reasoning."""
+
+_HOMEPAGE = "https://convex.mpi-inf.mpg.de"
+
+_LICENSE = "MIT License"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "http://qa.mpi-inf.mpg.de/convex/"
+_URLs = {
+    "train": _URL + "/ConvQuestions_train.zip",
+    "dev": _URL + "/ConvQuestions_dev.zip",
+    "test": _URL + "/ConvQuestions_test.zip",
+}
```

Comment:
there is already a "/" at the end of _URL

```suggestion
_URLs = {
    "train": _URL + "ConvQuestions_train.zip",
    "dev": _URL + "ConvQuestions_dev.zip",
    "test": _URL + "ConvQuestions_test.zip",
}
```